domains/certificates/chart/values.yaml

```diff
@@ -1,6 +1,9 @@
 api:
   replicaCount: 1
   image: {}
+  env:
+    MOCKMASTERDATA__JSONFILEPATH:
+    DATASYNC__URL:
```

Comment:
These should be removed, as eo-base will overwrite them. Therfore, they should be added over there

Reply:
removed in a893442